shader/fine.wgsl

```diff
@@ -251,7 +251,7 @@ fn main(
                 for (var i = 0u; i < PIXELS_PER_THREAD; i += 1u) {
                     let my_xy = vec2(xy.x + f32(i), xy.y);
                     // TODO: can hoist y, but for now stick to the GLSL version
-                    let xy_xformed = rad.matrx.xz * my_xy.x + rad.matrx.yw * my_xy.y - rad.xlat;
+                    let xy_xformed = rad.matrx.xy * my_xy.x + rad.matrx.zw * my_xy.y - rad.xlat;
```

Comment:
When I tested this change locally as is it didn't fix the broken rotation for me on the `blend_grid` test (you can reproduce it with the new Q/E controls that @DJMcNab added).

However I was able to fix it by reverting your change to fine.wgsl back to the original and applying @DJMcNab's suggestion on zulip to change [shaders/draw_leaf.wgsl#153](https://github.com/linebender/vello/blob/main/shader/draw_leaf.wgsl#L153) to:
```
let inv_mat = inv_det * vec4(matrx.w, matrx.y, matrx.z, matrx.x);
```

Reply:
The fix in this PR comes from experimenting with radial gradient rotation in general.

I used this codepen as reference: https://codepen.io/dfrgy/pen/KKxMpWj and added a corresponding example to the brush transform test scene. Without these changes, the transform is very wonky.

The original inverse calculation is algebraically correct so I'm assuming the problem is elsewhere but I haven't been able to track down the source of the reversed rotation direction in the blend grid. I'll keep investigating.

Reply:
Looks like the inverse of the translation components was incorrect. This seems to be working for all cases now.

Reply:
Or not...

Reply:
Ok, good to go :)